src/components/BulkMetadataForm/BulkMetadataFormStyles.ts

```diff
@@ -0,0 +1,89 @@
+import styled from 'styled-components';
+import { themeSpacing } from '@amsterdam/asc-ui';
+
+const BulkMetadataFormStyles = styled.div`
+	input[type='text'] {
+		margin: ${themeSpacing(3, 0, 6)};
+	}
+
+	.MuiGrid-item {
+		text-align: center;
+	}
+
+	.MuiGrid-spacing-xs-2 div:nth-of-type(2) {
+		div:first-of-type {
+			width: 100%;
+		}
+	}
+
+	.MuiCardContent-root .MuiGrid-container .MuiGrid-item .MuiGrid-container > div:first-of-type {
+		flex-grow: 2;
+	}
+
+	.MuiCardContent-root .MuiGrid-container .MuiGrid-item .MuiGrid-container > div:nth-of-type(2) > label > span {
+		width: 100%;
+	}
+
+	.MuiGrid-spacing-xs-2 {
+		margin: ${themeSpacing(0, 2)} !important;
```

Comment:
I don't believe `!important` is necessary here. Can you find a more appropriate method of achieving the required specificity?

Reply:
You're correct. !important is not necessary, removed.